src/scrape_up/amazon/products.py

```diff
@@ -191,3 +191,28 @@ def customer_review(self):
                 "data": None,
                 "message": f"Unable to fetch product review",
             }
+
+    def get_availability(self, soup):
+        """
```

Comment:
remove this `soup` argument, as it will prompt user also to provide while making a call.

Reply:
ok i am commiting